builtin/game/item.lua

```diff
@@ -335,6 +335,10 @@ function core.item_place(itemstack, placer, pointed_thing, param2)
 end
 
 function core.item_drop(itemstack, dropper, pos)
+	if math.abs(pos.x) > 30910 or math.abs(pos.y) > 30910 or math.abs(pos.z) > 30910 then
+		core.log("error", "itemdrop called outside of safe map bounds! @"..core.pos_to_string(pos))
```

Comment:
Isn't that writing the entities raw positions with a long floating ? Perhaps you should use `vector.round()` for trim it.